docs/content/kv_commands.md

```diff
@@ -159,15 +227,19 @@ The schema below is the full schema for key-value entries uploaded via the bulk
 
 If both `expiration` and `expiration_ttl` are specified for a given key, the API will prefer `expiration_ttl`.
 
+The `put` command also takes an optional `--env` [environment](./environments) argument.
+
 #### Usage
 
 ```sh
-$ wrangler kv:bulk put f7b02e7fc70443149ac906dd81ec1791 allthethingsupload.json
+$ wrangler kv:bulk put --binding=MY_NAMESPACE allthethingsupload.json
```

Comment:
I think it would be nice to have a sample json here with the expected format

I initially made a json like this:

```json
[{
    "four": "five",
    "six": "seven"
}]
```

```console
$ wrangler kv:bulk put -b OKV mykeys.json
Error: missing field `key` at line 4 column 1
```

This error would be better if it told you the schema to use instead of making you guess.

```json
[{
    "key": "five",
    "value": "seven"
},
{
    "key": "six",
    "value": "eight"
}]
```

It works fine once you get that

```console
wrangler kv:bulk put -b OKV mykeys.json
✨  Success
```

Reply:
we do have an example of this earlier in this section :)

Reply:
Ah! missed that - still would be good to have the format explained in the error message itself